src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int indexOfColor = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[indexOfColor];
+        return color.toString();
```

Comment:
Use the 'name()' method instead of 'toString()' to get the String representation of the enum. This ensures that the actual name of the enum constant is returned even if the 'toString()' method is overridden.